packages/authentication/src/decorator.ts

```diff
@@ -3,16 +3,10 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+import {AuthenticationMetadata} from '@loopback/core';
```

Comment:
+1, auth stuff should stay with auth component that is registered with core. The only thing about auth that should be normalized, is probably the binding keys used for LB to recognize it (or the component adhere's to the chosen interface and registered via app.component, etc).